### DISCUSSION TITLE
Dependency-Track 4.4.0: Github Advisory API token?

### DISCUSSION THREAD

**Discussion by verzada** [DependencyTrack/dependency-track#1407] [Q&A] [answered]
What is the api token for github advisory in Dependency Track? How do I go about creating the token? 
OSS Index and other vulnerability sources has been straight forward, this one has so far eluded me.

Looking at my profile at github, I can only have the options to create SSH or GPG.
Googling 'Github Advisory api token' does not yield result either (as far as I can see).

**Comment by stevespringett** [comment, marked as answer]
In GitHub, go to: 

Settings → Developer settings → Personal access tokens

**Reply by verzada** [reply]
Thank you so much 😃
I didn't know about developer settings at all, the settings isn't hidden but it's not really at the top either 🤷🏼‍♂️

**Comment by falcanta-akamai** [comment]
Where would you put the GHSA API token in dtrack? additionally what would be the scope for that specific API token? I did see this but it doesn't say anything about GHSA scope: https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps

**Reply by stevespringett** [reply]
The scope can be empty. Any authenticated user has access to GitHub Advisories. 

<img width="1253" alt="Screen Shot 2022-02-23 at 4 52 55 PM" src="https://user-images.githubusercontent.com/3878933/155423057-4b4ea2c7-74ca-42d5-8a93-3172bd15bb9d.png">

**Reply by falcanta-akamai** [reply]
Ok, understood. I don't see the vulnerability sources in my instance. I'm currently running dtrack version 4.3.6 for my setup. 
![Screen Shot 2022-02-24 at 8 50 03 AM](https://user-images.githubusercontent.com/90790161/155536575-4ccc7821-229b-45c2-840c-59ea260a4866.png)

**Comment by verzada** [comment]
Yeah, should have pointed out in the issue, it's about 4.4.1 and 4.4.0, not earlier versions.
